pkg/cmd/run/root.go

```diff
@@ -70,7 +70,7 @@ var runCmd = &cobra.Command{
 			utils.CheckErr(err)
 		}
 
-		dash, err := dashboard.New(proj, envMap, true)
```

Comment:
Is this removing the changes from the auto open browser PR?

Reply:
yep, that should not be in there. Fixed